tests/unit/session-stores/adaptive-test.js

```diff
@@ -6,13 +6,18 @@ import Adaptive from 'ember-simple-auth/session-stores/adaptive';
 import itBehavesLikeAStore from './shared/store-behavior';
 import itBehavesLikeACookieStore from './shared/cookie-store-behavior';
 import FakeCookieService from '../../helpers/fake-cookie-service';
+import Configuration from 'ember-simple-auth/configuration';
 
 const { assign: emberAssign, merge, run } = Ember;
 const assign = emberAssign || merge;
 
 describe('AdaptiveStore', () => {
   let store;
 
+  beforeEach(function() {
+    Configuration.load({ baseURL: '/' });
```

Comment:
yes, that's probably a better option then to globally force this setting to all tests